R/update-news.R

```diff
@@ -24,7 +27,10 @@ collect_news <- function(messages) {
     }
   }
 
-  cli_alert_success("Found {.field {length(message_items)}} NEWS-worthy entries.")
+  if (fledge_chatty()) {
+    cli_alert_success("Found {.field {length(message_items)}} NEWS-worthy entries.")
```

Comment:
btw I tried wrapping `cli_alert()` into another function but it was a mess because of the glue-like stuff in `cli_alert()` so it was simpler to use the conditional.